packages/jest-transform/src/__tests__/script_transformer.test.js

```diff
@@ -213,52 +209,46 @@ describe('ScriptTransformer', () => {
 
   it('transforms a file properly', () => {
     const scriptTransformer = new ScriptTransformer(config);
-    const response = scriptTransformer.transform(
+    const transformedBananaWithCoverage = scriptTransformer.transform(
       '/fruits/banana.js',
       makeGlobalConfig({collectCoverage: true}),
-    ).script;
+    );
 
-    expect(response instanceof vm.Script).toBe(true);
-    expect(wrap(vm.Script.mock.calls[0][0])).toMatchSnapshot();
+    expect(wrap(transformedBananaWithCoverage.code)).toMatchSnapshot();
 
     // no-cache case
     expect(fs.readFileSync).toHaveBeenCalledTimes(1);
     expect(fs.readFileSync).toBeCalledWith('/fruits/banana.js', 'utf8');
 
     // in-memory cache
-    const response2 = scriptTransformer.transform(
+    const transformedBananaWithCoverageAgain = scriptTransformer.transform(
       '/fruits/banana.js',
       makeGlobalConfig({collectCoverage: true}),
-    ).script;
-    expect(response2).toBe(response);
-
-    scriptTransformer.transform(
-      '/fruits/kiwi.js',
-      makeGlobalConfig({collectCoverage: true}),
     );
-    const snapshot = vm.Script.mock.calls[1][0];
-    expect(wrap(snapshot)).toMatchSnapshot();
+    expect(transformedBananaWithCoverageAgain).toBe(
+      transformedBananaWithCoverage,
+    );
 
-    scriptTransformer.transform(
+    const transformedKiwiWithCoverage = scriptTransformer.transform(
       '/fruits/kiwi.js',
       makeGlobalConfig({collectCoverage: true}),
     );
+    expect(wrap(transformedKiwiWithCoverage.code)).toMatchSnapshot();
 
-    expect(vm.Script.mock.calls[0][0]).not.toEqual(snapshot);
-    expect(vm.Script.mock.calls[0][0]).not.toMatch(/instrumented kiwi/);
+    expect(transformedBananaWithCoverage.code).not.toEqual(
+      transformedKiwiWithCoverage.code,
+    );
+    expect(transformedBananaWithCoverage.code).not.toMatch(/instrumented kiwi/);
 
     // If we disable coverage, we get a different result.
-    scriptTransformer.transform(
+    const transformedKiwiWithoutCoverage = scriptTransformer.transform(
       '/fruits/kiwi.js',
       makeGlobalConfig({collectCoverage: false}),
     );
-    expect(vm.Script.mock.calls[1][0]).toEqual(snapshot);
 
-    scriptTransformer.transform('/fruits/banana.js', {
-      // to make sure jest isn't declared twice
```

Comment:
this was broken anyways - if we declared it and then removed it here, we would still inject it from `jest-runtime`